src/bin/electrs.rs

```diff
@@ -153,8 +153,6 @@ fn main_() {
     }
 }
 
-#[cfg(not(any(feature = "otlp-tracing", feature = "no-otlp-tracing")))]
-compile_error!("Must enable one of the 'otlp-tracing' or 'no-otlp-tracing' features");
```

Comment:
Why was this check removed? If we're keeping the features it would be better to cover both failure cases.

Also, I would place this in `lib.rs`.

Reply:
I'll move it. Regarding that removed condition, I just couldn't trigger it. Always the second compile check was triggered

Reply:
It should be triggered if you use `--no-default-features` without enabling anything, doesn't it?

Reply:
You're right it is. I missed it because it also causes a flood of 
"error: cannot determine resolution for the attribute macro `instrument`" errors as expected. I added this check too.